packages/web/hooks/use-export.ts

```
@@ -0,0 +1,590 @@
+          const singleResult = await exportService.export(
+            content,
+            { format: exportOptions.format },
+            metadata
+          )
+
+          if (!singleResult.success) {
+            throw new Error(singleResult.error || 'Export failed')
+          }
+
+          // Convert to ExportResult type
+          let content = ''
```

Biome (2.1.2)

[error] Shouldn't redeclare 'content'. Consider to delete it or rename it.

'content' is defined here:


(lint/suspicious/noRedeclare)